.prettierrc.json

```diff
@@ -1,4 +1,8 @@
 {
+	"useTabs": true,
+	"singleQuote": false,
+	"semi": true,
+	"endOfLine": "lf",
```

Comment:
Some of these options are duplicates since Prettier respects the `.editorconfig` file settings, your choice if you want to be extra specific here.